### DISCUSSION TITLE
UTMStackAgent ERROR: Error sending logs to Log Auth Proxy: rpc error: code = Internal desc = grpc: error while marshaling: string field contains invalid UTF-8

### DISCUSSION THREAD

**Discussion by balajiuma2736** [utmstack/UTMStack#323] [Q&A] [answered]
UTMStackAgent ERROR: Error sending logs to Log Auth Proxy: rpc error: code = Internal desc = grpc: error while marshaling: string field contains invalid UTF-8

**Comment by osmontero** [comment, marked as answer]
Linked to issue: https://github.com/utmstack/UTMStack/issues/329
